_data/anubhav823/anubhav823.md

```diff
@@ -0,0 +1,5 @@
+name: Anubhav Jain
+institution: Kalyani Government Engineering College
+profile_pic: 30366878.jpg
+quote: At least the wifi is working
+github_user: anubhav823
```

Comment:
Your markdown is not correct. Please check [instructions](https://github.com/education/graduation#how-to-join-the-class-of-2020-closed-)

Reply:
plz reviee #593 also